src/index.jsx

```diff
@@ -304,23 +322,37 @@ class LazyLoad extends Component {
     }
   }
 
+  setRef(element) {
+    if (element) {
+      this.ref = element;
+    }
+  }
+
   render() {
-    return this.visible ?
-      this.props.children :
-      this.props.placeholder ?
-        this.props.placeholder :
-        <div style={{ height: this.props.height }} className="lazyload-placeholder" />;
-    // <span ref={this.setRef} className="lazyload-custom-placeholder">
-    //   {this.props.placeholder}
-    // </span> :
-    // <div ref={this.setRef} style={{ height: this.props.height }} className="lazyload-placeholder" />;
+    return (
+      <span className="lazyload-wrapper" ref={this.setRef}>
```

Comment:
what was the reason for adding this container?

Reply:
previously we used findDomNode to get reference to the lazyloaded component's html container element, to remove that we have used ref, do suggest if you think there is any other way to implement the same.